README.md

```diff
@@ -187,6 +192,8 @@ You should never merge your own pull requests. A PR gives the rest of your team
 
   <img src="images/pull-request-tab.png" width="500" height="auto" alt="pull request tab button">    
 
+Now your remote repo looks like this: </br>
```

Comment:
i dont think the `</br>` doing anything :sweat_smile: 
either way i cant see a space between the image and this line